tests/test_blocks.py

```diff
@@ -366,3 +366,29 @@ def test_callout():
         '\n'
         'Children\n'
     )
+
+
+def test_synced_block():
```

Comment:
I'd split this out into two tests that are more granular, that way the test names (and any failures) can more quickly be honed into.

E.g., `test_synced_block_shared` and `test_scyned_block_unshared`.